sdk/network/Azure.ResourceManager.Network/samples/Generated/Samples/Sample_LoadBalancerResource.cs

```diff
@@ -145,7 +145,7 @@ public async Task Update_UpdateLoadBalancerTags()
         public async Task MigrateToIPBased_MigrateLoadBalancerToIPBased()
         {
             // Generated from example definition: specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/examples/MigrateLoadBalancerToIPBased.json
-            // this example is just showing the usage of "LoadBalancers_MigrateToIPBased" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "LoadBalancers_MigrateToIpBased" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `IP`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/loadBalancer.json#L1599